### ISSUE TITLE
fatally missing registry entries

### ISSUE THREAD

**Issue by axin0452** [codetaylor/pyrotech#275]
I don't know how can I specifically describe this problem,But this problem does exist,I troubleshoot each mod,This prompt only appears when pyrotech is present,I don't know what else I can do for this, hope you can help me!

    Minecraft: 1.12.2
    SpongeAPI: 7.1.0-ea50f0c2
    Sponge: 1.12.2-7.1.9
    SpongeForge: 1.12.2-2838-7.1.9
    Minecraft Forge: 14.23.5.2847
    JVM: 1.8.0_131/64-bit (Oracle Corporation)
    OS: Windows 10 (10.0/amd64)
   dropt-1.12.2-1.16.1
   pyrotech-1.12.2-1.4.20
   athenaeum-1.12.2-1.17.4

https://paste.ubuntu.com/p/4spDhRx7GS/

**Comment by codetaylor** [comment]
After reading several threads about this issue on minecraftforge.net, my best guess is that there is a discrepancy between the modpack on the server and the modpack on the client. The general consensus of these threads seems to be to make sure you have the same mods on both the server and client and that all of the configs match.

https://www.google.com/search?q=site%3Aminecraftforge.net+fatally+missing+registry+entries

I also noticed this:
```
[21:09:37] [Server thread/ERROR] [pyrotech.ModuleTechBloomery]: Missing ore compat lang key for: oreCopper
```
It could be related, I'm not sure.

See this for more info on the ore compatibility system: https://pyrotech.readthedocs.io/en/latest/orecompat/

**Comment by axin0452** [comment]
@codetaylor 
When I try to delete config \ pyrotech, the game returns to normal, but whenever I add a mod to the server, it will cause this problem. Even if it is a repair mod, it can only be repaired by deleting config \ pyrotech. Is this normal?

**Comment by codetaylor** [comment]
> Is this normal?
I don't think so. This is the first I'm hearing of it and I'm not really sure why it's happening.

**Comment by codetaylor** [comment]
If I had to make a guess, I would say it's something to do with the auto-generated files that generate the first time Pt is loaded. If Pt is loaded and generates content, then the mod it generated content from is removed, it might have a problem. Then when the configs are deleted, it generates new ones based on the current content and all's fine again. I don't really know, it's just a stab in the dark.

**Comment by axin0452** [comment]
Just now I changed serve.properties/level-type to quark_realistic and deleted the world folder to regenerate the world. This problem happened again. I really have no clue and I do n’t know what I did wrong.

**Comment by axin0452** [comment]
@codetaylor 
I made a lot of attempts just now, removing all the plugins and closing waterfall, there is no way to solve it, as long as there are any changes in the mods folder, this problem will occur. What information do I need to provide you to get more help?

**Comment by codetaylor** [comment]
I don't know how to help you with this.
